test/jdk/sun/security/provider/acvp/Launcher.java

```diff
@@ -37,12 +43,14 @@
 
 /// This test runs on `internalProjection.json`-style files generated
 /// by NIST's ACVP Server. See [https://github.com/usnistgov/ACVP-Server].
+/// Download zip archive, for instance -
```

Comment:
The 1st sentence says `internalProjection.json` but the 2nd says a zip archive. Maybe we can mention the relation. For example,
```
/// This test runs on `internalProjection.json`-style files generated by NIST's
/// ACVP Server ([GitHub repository](https://github.com/usnistgov/ACVP-Server)).
/// These files are included in ZIP archives available under the
/// [tags section](https://github.com/usnistgov/ACVP-Server/tags)
/// of the repository.
```